README.md

```diff
@@ -81,3 +81,7 @@ class MyChat implements MessageComponentInterface {
     conn.onmessage = function(e) { console.log(e.data); };
     conn.onopen = function(e) { conn.send('Hello Me!'); };
 ```
+
+[GA Image]: https://github.com/ratchetphp/Ratchet/workflows/CI/badge.svg
```

Comment:
Should this be .github/workflows?

Reply:
Not sure what you mean but it links to the github badge url for workflow named `CI`. Above url is correct.

https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge

Reply:
ah, my bad